xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-6.33.0
+6.33.1
```

Comment:
Current master is 6.33.1, bumped by #8654.

https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/json-rpc/schema/version.txt